test/tests/functional/pbs_hook_set_nonexist.py

```diff
@@ -0,0 +1,55 @@
+# coding: utf-8
```

Comment:
Isn't there a hook test suite to which this can be added, instead of a new suite?

Reply:
Will check and try to move to existing testsuite if any.